src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -150,7 +150,11 @@ public function getDefinition(): FixerDefinitionInterface
                 new CodeSample($code),
                 new CodeSample($code, ['align' => self::ALIGN_VERTICAL]),
                 new CodeSample($code, ['align' => self::ALIGN_LEFT]),
-            ]
+            ],
+            'Accepts a subset of tags with name, tags with method, and custom tags.
+tags with name: `[\''.implode('\', \'', self::TAGS_WITH_NAME).'\']`
+tags with method: `[\''.implode('\', \'', self::TAGS_WITH_METHOD_SIGNATURE).'\']`
+custom tags: custom tag name with description e.g. `@tag <desc>`.'
```

Comment:
I would move it to option's description (`$tags = new FixerOptionBuilder('tags', 'The tags that should be aligned.');`, line 202), with slight changes like this:

```php
        $tags = new FixerOptionBuilder('tags', 'The tags that should be aligned. Allowed values:
- tags with name: `[\''.implode('\', \'', self::TAGS_WITH_NAME).'\']`
- tags with method signature: `[\''.implode('\', \'', self::TAGS_WITH_METHOD_SIGNATURE).'\']`
- any custom tag with description e.g. `@tag <desc>`');
```

Reply:
Should end with a dot 😅.